gui/src/app/view/receive.rs

```diff
@@ -115,3 +132,53 @@ pub fn receive<'a>(
         .spacing(20)
         .into()
 }
+
+pub fn verify_address_modal<'a>(
+    warning: Option<&Error>,
+    hws: &'a [HardwareWallet],
+    chosen_hws: &HashSet<Fingerprint>,
+    address: &Address,
+    derivation_index: &ChildNumber,
+) -> Element<'a, Message> {
+    Column::new()
+        .push_maybe(warning.map(|w| warn(Some(w))))
+        .push(card::simple(
+            Column::new()
+                .push(
+                    Column::new()
+                        .push(
+                            Container::new(
+                                Row::new()
+                                    .push(text(address.to_string()).bold())
+                                    .push(text(format!("#{}", derivation_index)))
```

Comment:
It's not exactly clear this refers to. What do you think of changing the display to:
- Address: `b1rt....`
- Derivation index: `XXX`